### PR TITLE
[ci] add window + macosx pool to raycicmd

### PR DESCRIPTION
**Pull request by can-anyscale**
As title, add window + macosx pool

### CODE REVIEW

File: raycicmd/config.go [hunk 1]
```diff
@@ -165,13 +165,16 @@ var branchPipelineConfig = &config{
...
-		"gpu-large": "gpu_large_runner_queue_branch",
-		"trainium":  "trainium_runner_queue_branch",
+		"default":     "runner_queue_small_branch",
+		"small":       "runner_queue_small_branch",
+		"medium":      "runner_queue_medium_branch",
+		"large":       "runner_queue_branch",
+		"gpu":         "gpu_runner_queue_branch",
+		"gpu-large":   "gpu_large_runner_queue_branch",
+		"trainium":    "trainium_runner_queue_branch",
+		"window":      "windows_queue_branch",
```
Comment (aslonnie) [inline raycicmd/config.go:175]: could you change `window` to `win` or `windows`?

Comment (aslonnie) [reply raycicmd/config.go:175]: and at this point I think we should add step skipping based on jobenv / instance types now..

**Comment by can-anyscale** [comment]
updated to windows \o/

**Comment by can-anyscale** [comment]
fmt \o/

**Review by aslonnie — APPROVED** [review]

**Pull request merged**